demo/codegen/Readme.md

```diff
@@ -0,0 +1,14 @@
+# demo/
+
+## Warning
+These examples have test case dependencies, think carefuly before changing them
```

Comment:
leave this in demo/ (It says demo/ in the title)

Reply:
That's why I had left it in demo in the 1st place, will roll back the change.